queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -36,58 +43,87 @@ def get_next_corp_num(business_type: str, application: Flask):
 
 def update_business_info(corp_num: str, business: Business, business_info: Dict, filing: Dict, filing_rec: Filing):
     """Format and update the business entity from incorporation filing."""
-    if corp_num and business and business_info:
+    if corp_num and business and business_info and filing and filing_rec:
+        legal_name = business_info.get('legalName', None)
         business.identifier = corp_num
-        business.legal_name = filing.get('nameRequest', {}).get('legalName', None)
-        business.legal_type = filing.get('nameRequest', {}). get('legalType', None)
+        business.legal_name = legal_name if legal_name else corp_num
```

Comment:
If it is a numbered company, I think the legal name should have B.C. LTD appended to the corp num

Reply:
thanks!